runtime/src/serde_snapshot.rs

```diff
@@ -42,18 +40,20 @@ use {
         result::Result,
         sync::{
             atomic::{AtomicUsize, Ordering},
-            Arc, RwLock,
+            Arc,
         },
         thread::Builder,
     },
-    storage::{SerializableStorage, SerializedAppendVecId},
+    storage::SerializableStorage,
 };
 
 mod newer;
 mod storage;
 mod tests;
 mod utils;
 
+use crate::snapshot_utils::StorageAndNextAppendVecId;
```

Comment:
this belongs in the 'use' at the top of the file.

Reply:
'use's

Reply:
thanks! usually format combines these for me, but didn't for some reason.